VideoPlusChat/src/test/java/com/example/videochatcomposesample/ExampleUnitTest.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.videochatcomposesample
```

Comment:
Please, delete this dummy test as it doesn't provide any value right now and is just noise.